deepmd/pd/model/descriptor/repformers.py

```
@@ -0,0 +1,576 @@
+        if comm_dict is None:
+            if paddle.in_dynamic_mode():
+                assert isinstance(extended_atype_embd, paddle.Tensor)  # for jit
+            atype_embd = extended_atype_embd[:, :nloc, :]
+            if paddle.in_dynamic_mode():
+                assert list(atype_embd.shape) == [nframes, nloc, self.g1_dim]
+        else:
+            atype_embd = extended_atype_embd
+            if paddle.in_dynamic_mode():
+                assert isinstance(atype_embd, paddle.Tensor)  # for jit
+        g1 = self.act(atype_embd)
+        ng1 = g1.shape[-1]
```

Ruff (0.8.0)

Local variable `ng1` is assigned to but never used

Remove assignment to unused variable `ng1`

(F841)